Add Windows instructions to README.md (#34)
signalfx/signalfx-dotnet-tracing: README.md

# SignalFx Tracing Library for .NET

The SignalFx Tracing Library for .NET provides an
OpenTracing-compatible tracer and automatically configured instrumentations
for popular .NET libraries and frameworks.  It supports .NET Core 2.0+ on

> Linux and Windows and .NET Framework 4.5+ on Windows.

Reviewer: Not sure official support for framework is intended atm.  Instrumentations are provided under beta status?

Author: Yes, we are adding Framework support too. The beta status for the instrumentation is mentioned below.

    $ dotnet run
    ```

### Windows

> 1. Install the CLR Profiler using the installers (MSI files) from
> [latest release](https://github.com/signalfx/signalfx-dotnet-tracing/releases/latest),
> choose the installer (x64 or x86) according to the architecture of the application
> to be instrumented.

Reviewer 2: ```suggestion
1. Install the CLR Profiler using an installer file (`.msi` file) from the
[latest release](https://github.com/signalfx/signalfx-dotnet-tracing/releases/latest).
Choose the installer (x64 or x86) according to the architecture of the application
you're instrumenting.
```

> 2. Configure the required environment variables:

Reviewer 2: Steps 2-4 involve setting environment variables. For step 2, can we be more specific about which environment variables a user is setting?

    - For .NET Framework applications:
    ```batch
    set COR_ENABLE_PROFILING=1
    set COR_PROFILER={B4C89B0F-9908-4F73-9F59-0D77C5A06874} 
    ```
   - For .NET Core applications:
   ```batch
   set CORECLR_ENABLE_PROFILING=1
   set CORECLR_PROFILER={B4C89B0F-9908-4F73-9F59-0D77C5A06874}
   ```
3. Set the service name:
   ```batch
   set SIGNALFX_SERVICE_NAME=MyServiceName
   ```
4. Set the endpoint of a Smart Agent or OpenTelemetry Collector:
   ```batch
   set SIGNALFX_ENDPOINT_URL='http://<YourSmartAgentOrCollector>:9080/v1/trace'
   ```

> 5. Re-start your application ensuring that all environment variables are properly

Reviewer 2: ```suggestion
5. Restart your application ensuring that all environment variables are properly
```

Reviewer 2: What should a user do to ensure that all environment variables are properly configured?

Author: AFAIK Windows doesn't have a direct way to do this I will suggest a tool that can help with that - the tool is well-known for Windows users.
